src/loaders/BlueprintLoader.java

```diff
@@ -36,9 +36,15 @@
 	}
 
 	public static int extractEID(String input) {
+		// expected syntax is {eID=2}
+		// indexOf("eID=") has to return at least 1 if correct syntax
```

Comment:
Actually indexOf("eID=") should return at least 2. The {eID=x} part has to have a component name in front of it. And for that to exist it has to have at least one character in it. So component name length + 1 (for the "{") which has to be at least 2.